src/Export/ProductService.php

```diff
@@ -26,33 +28,45 @@ class ProductService
 {
     public const CONTAINER_ID = 'fin_search.product_service';
 
+    /** @var Config|null */
+    private $config;
+
     /** @var ContainerInterface */
     private $container;
 
     /** @var SalesChannelContext|null */
     private $salesChannelContext;
 
-    public function __construct(ContainerInterface $container, ?SalesChannelContext $salesChannelContext = null)
-    {
+    public function __construct(
+        ContainerInterface $container,
+        ?SalesChannelContext $salesChannelContext = null,
+        ?Config $config = null
```

Comment:
This is really important, as otherwise this wouldn't be backwards compatible :+1: